### ISSUE TITLE
Reflector not updating mirrors on install.

### ISSUE THREAD

**Issue by 5ouls3dge** [archlinux/archinstall#2338]
Hi after running the script. It finishes formatting then say it's Waiting for automatic mirror selection (reflector) to complete.". This never finishes.

Journal says it fails to start reflector.

EndeavourOS also has issues updating the mirror list. And manual copy paste of archlinux list into pacman.d/mirror list solves it. But when I add a custom mirror in archinstall I still get the error. Any workaround

**Comment by someqst** [comment]
have same problem

**Comment by dewaltz** [comment]
Did custom mirror work for you?

**Comment by dewaltz** [comment]
Seems it will always push to reflector in the script. Which means if reflector faults then there is no fallback to add a mirror?

**Comment by GustavoTZ14** [comment]
hi does anyone know if the reflector problem has already been solved, i went to install arch and i couldn't either.

**Comment by bknightInfo** [comment]
A work around until it is fixed is 
systemctl mask reflector.service

**Comment by dewaltz** [comment]
Thank you... And don't forget to edit the list.... Nano /etc/pacman.d/mirrorlist

Remove all the entries or RM mirror list then nano mirror list and add a server Server = https://.... From https://archlinux.org/mirrors/status/ choose a successful syncing scroll down then find that in: https://archlinux.org/mirrorlist/

**Comment by dugonzal** [comment]
presento el mismo problema fallan las claves del espejo, no se realmente como solucionarlo

**Comment by dugonzal** [comment]
lo solucione, el problema son los servidores espejos, basicamente borre el /etc/pacman.d/mirrolist y agregue un servidor espejo que este en funcionamiento por ejemplo " https://nox.panibrez.com/archlinux/$repo/os/$arch" luego actualizo las keys con pacman-key --init; pacman-key --populate y luego instale como de costrubre y sin problemas

**Comment by Dlinuigh** [comment]
I think new system should just use the live system mirror settings, for example, copy the mirrorlist to new system will be ok for me.

**Comment by someqst** [comment]
Move to ISO archlinux-2023.11.01-x86_64 and do things below. Its helped me.

Maybe there is a package that is conflicting, as this error occurred using the archlinux-2024.01.01-x86_64 ISO and then I tested using the archlinux-2023.12.01-x86_64 ISO.
So I took the ISO archlinux-2023.11.01-x86_64 and updated archinstall to the latest version, at the time of this comment it is version 2.7.1-1. The python package was version 3.11.5-2, which was updated to 2.7.1-1 when updating the archinstall package.

With this version of ISO, 2023.11.01, the installation went like a glove.
Detail, in this version, you must also update the archlinux-keyring package to avoid the PGP Key error.

Listen to the detail, as the installation gave a PGP Key error, I had to delete the partitions again with cfdisk and call archinstall again. Without doing this, it gave the same error (Unable to determine new UUID) as the other ISOs
